tests/test_deprecations.py

```diff
@@ -139,11 +139,11 @@ def test_action(
         parser = ArgumentParser()
         parser.add_argument(
             "--foo",
-            action=deprecated.action("2.0", "3.0", _StoreTrueAction),
+            action=deprecated.action("2.0", "3.0", _StoreAction),
         )
```

Comment:
I think we should explicitly test this as a positional and leave the `--foo` as it was:

```suggestion
        )
        parser.add_arguemnt(
            "bar",
            action=deprecated.action("2.0", "3.0", _StoreAction),
        )
```